test/unittest/unittest_nntrainer_layers.cpp

```diff
@@ -572,6 +572,54 @@ TEST(nntrainer_Conv2DLayer, forwarding_02_p) {
   }
 }
 
+/**
+ * @brief Convolution 2D Layer
+ */
+TEST(nntrainer_Conv2D, backwarding_01_p) {
+  int status = ML_ERROR_NONE;
+  nntrainer::Conv2DLayer layer;
+  std::vector<std::string> input_str;
+  nntrainer::TensorDim previous_dim;
+  previous_dim.setTensorDim("1:3:7:7");
+
+  input_str.push_back("input_shape=1:3:7:7");
+  input_str.push_back("bias_zero=true");
+  input_str.push_back("activation=sigmoid");
+  input_str.push_back("weight_decay=l2norm");
+  input_str.push_back("weight_decay_lambda = 0.005");
+  input_str.push_back("weight_ini=xavier_uniform");
+  input_str.push_back("filter=2");
+  input_str.push_back("kernel_size= 3,3");
+  input_str.push_back("stride=1, 1");
+  input_str.push_back("padding=0,0");
+
+  status = layer.setProperty(input_str);
+  EXPECT_EQ(status, ML_ERROR_NONE);
+  layer.setInputDimension(previous_dim);
+
+  status = layer.initialize(true);
+  EXPECT_EQ(status, ML_ERROR_NONE);
+
+  nntrainer::Tensor in(1, 3, 7, 7);
+  nntrainer::Tensor out;
+  nntrainer::Tensor derivatives(1, 2, 5, 5);
+  float golden[50] = {0.25, 0.5, 0.5, 0.5, 0.25, 0.5,  1.0, 1.0, 1.0, 0.5,
```

Comment:
can we rename this from `golden` to `sample_derivative`. 
Golden sounds like derivative is going to be matched with it, but it is just a sample value to check that code runs smoothly.

Reply:
fixed. thanks.